### PR TITLE
FRONT-6845 update dependencies to fix snyk vulns

### PR DESCRIPTION
**Pull request by stuart-gill**
[FRONT-6845](https://scalyr.myjetbrains.com/youtrack/issue/FRONT-6845) Snyk Reported Vulnerability in scalyr-java-client https://app.snyk.io/org/scalyr-org/project/bad7bcba-9fb3-43e1-9a9e-92bac56f281a

### CODE REVIEW

**Comment by stuart-gill** [comment]
I updated the two relevant dependencies and ran tests locally and everything passed... CI passed also... should be good to go.

File: pom.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>19.0</version>
+            <version>30.0-android</version>
```
Comment (pulyaevskiy) [inline pom.xml:86]: Is this the Snyk-suggested version? Seems a bit odd to use android flavor here.

Comment (pulyaevskiy) [reply pom.xml:86]: Oh I think you can use `30.0-jre` (according to Snyk report).

Comment (stuart-gill) [reply pom.xml:86]: Yes, it is the Snyk suggested version, and I wondered about the android bit. I'll change to 31.0.1-jre

Comment (stuart-gill) [reply pom.xml]: all right, 30.0-jre worked fine, tests passed!

**Comment by stuart-gill** [comment]
Adding John as a reviewer because Anatoly doesn't have much experience with the Java Client. John, do you recommend any particular testing beyond unit tests and CI?

**Review by jjhart — COMMENTED** [review]
I believe we should also change the version of our package, up at line 6.  I'm not an expert in that process but I believe it's documented .. I think @djungyvr can provide pointers.

**Comment by djungyvr** [comment]
> but I believe it's documented

Yup! The process is [here](https://www.notion.so/scalyr/Deploying-to-Maven-Central-0d29b08df54b4202bc1ab3cbfc0e40b2#4935a56886b640e4b6962e024e09b246).

For this PR though I don't think any version changes in `pom.xml` are necessary since we do it as part of the release, so we can just save that for later.

**Comment by stuart-gill** [comment]
@djungyvr -- a few questions:

-  Should I do anything beyond running unit tests to make sure that updating these two dependencies won't break the client? 
- If not, would you approve this PR?
- What determines when a new version should be released? We've added the onlyUseSummaries and createSummaries timeseries flags to the client last month (in [PR 54](https://github.com/scalyr/Scalyr-Java-Client/pull/54)), so I could imagine that being a reason to release a new version.

**Review by djungyvr — APPROVED** [review]
Post-merge let's push a `-SNAPSHOT` version to maven and we'll try it out.

**Comment by djungyvr** [comment]
Actually my main concern was that the `httpclient` version update would affect our ability to connect to our servers but I just tested it locally and the logs I generated locally made its way to `prodeu` ([link](https://app.eu.scalyr.com/y/bPdAoMrQY5R))

So I think we should be ok to just push a new version.

**Pull request merged**